{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import glob\n",
    "import csv\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['lipid code set2-wounding.csv', 'set2_W1.csv', 'set2_W10.csv', 'set2_W100.csv', 'set2_W101.csv', 'set2_W102.csv', 'set2_W103.csv', 'set2_W104.csv', 'set2_W105.csv', 'set2_W106.csv', 'set2_W107.csv', 'set2_W108.csv', 'set2_W109.csv', 'set2_W11.csv', 'set2_W110.csv', 'set2_W111.csv', 'set2_W112.csv', 'set2_W113.csv', 'set2_W114.csv', 'set2_W115.csv', 'set2_W116.csv', 'set2_W117.csv', 'set2_W118.csv', 'set2_W119.csv', 'set2_W12.csv', 'set2_W120.csv', 'set2_W121.csv', 'set2_W122.csv', 'set2_W123.csv', 'set2_W124.csv', 'set2_W125.csv', 'set2_W126.csv', 'set2_W127.csv', 'set2_W128.csv', 'set2_W129.csv', 'set2_W13.csv', 'set2_W130.csv', 'set2_W131.csv', 'set2_W132.csv', 'set2_W133.csv', 'set2_W134.csv', 'set2_W135.csv', 'set2_W136.csv', 'set2_W137.csv', 'set2_W138.csv', 'set2_W139.csv', 'set2_W14.csv', 'set2_W140.csv', 'set2_W141.csv', 'set2_W142.csv', 'set2_W143.csv', 'set2_W144.csv', 'set2_W145.csv', 'set2_W146.csv', 'set2_W147.csv', 'set2_W148.csv', 'set2_W149.csv', 'set2_W15.csv', 'set2_W150.csv', 'set2_W151.csv', 'set2_W152.csv', 'set2_W153.csv', 'set2_W154.csv', 'set2_W155.csv', 'set2_W156.csv', 'set2_W157.csv', 'set2_W158.csv', 'set2_W159.csv', 'set2_W16.csv', 'set2_W160.csv', 'set2_W161.csv', 'set2_W162.csv', 'set2_W163.csv', 'set2_W164.csv', 'set2_W165.csv', 'set2_W166.csv', 'set2_W167.csv', 'set2_W168.csv', 'set2_W169.csv', 'set2_W17.csv', 'set2_W170.csv', 'set2_W171.csv', 'set2_W172.csv', 'set2_W173.csv', 'set2_W174.csv', 'set2_W175.csv', 'set2_W176.csv', 'set2_W177.csv', 'set2_W178.csv', 'set2_W179.csv', 'set2_W18.csv', 'set2_W180.csv', 'set2_W181.csv', 'set2_W182.csv', 'set2_W183.csv', 'set2_W184.csv', 'set2_W185.csv', 'set2_W186.csv', 'set2_W187.csv', 'set2_W188.csv', 'set2_W189.csv', 'set2_W19.csv', 'set2_W190.csv', 'set2_W191.csv', 'set2_W192.csv', 'set2_W193.csv', 'set2_W194.csv', 'set2_W195.csv', 'set2_W196.csv', 'set2_W197.csv', 'set2_W198.csv', 'set2_W199.csv', 'set2_W2.csv', 'set2_W20.csv', 'set2_W200.csv', 'set2_W201.csv', 'set2_W202.csv', 'set2_W203.csv', 'set2_W204.csv', 'set2_W205.csv', 'set2_W206.csv', 'set2_W207.csv', 'set2_W208.csv', 'set2_W209.csv', 'set2_W21.csv', 'set2_W210.csv', 'set2_W211.csv', 'set2_W212.csv', 'set2_W213.csv', 'set2_W214.csv', 'set2_W215.csv', 'set2_W216.csv', 'set2_W217.csv', 'set2_W218.csv', 'set2_W219.csv', 'set2_W22.csv', 'set2_W220.csv', 'set2_W221.csv', 'set2_W222.csv', 'set2_W223.csv', 'set2_W224.csv', 'set2_W225.csv', 'set2_W226.csv', 'set2_W227.csv', 'set2_W228.csv', 'set2_W229.csv', 'set2_W23.csv', 'set2_W230.csv', 'set2_W231.csv', 'set2_W232.csv', 'set2_W233.csv', 'set2_W234.csv', 'set2_W235.csv', 'set2_W236.csv', 'set2_W237.csv', 'set2_W238.csv', 'set2_W239.csv', 'set2_W24.csv', 'set2_W240.csv', 'set2_W241.csv', 'set2_W242.csv', 'set2_W243.csv', 'set2_W244.csv', 'set2_W245.csv', 'set2_W246.csv', 'set2_W247.csv', 'set2_W248.csv', 'set2_W249.csv', 'set2_W25.csv', 'set2_W250.csv', 'set2_W251.csv', 'set2_W252.csv', 'set2_W253.csv', 'set2_W254.csv', 'set2_W255.csv', 'set2_W256.csv', 'set2_W257.csv', 'set2_W258.csv', 'set2_W259.csv', 'set2_W26.csv', 'set2_W260.csv', 'set2_W261.csv', 'set2_W262.csv', 'set2_W263.csv', 'set2_W264.csv', 'set2_W265.csv', 'set2_W266.csv', 'set2_W267.csv', 'set2_W268.csv', 'set2_W269.csv', 'set2_W27.csv', 'set2_W270.csv', 'set2_W271.csv', 'set2_W272.csv', 'set2_W273.csv', 'set2_W274.csv', 'set2_W275.csv', 'set2_W276.csv', 'set2_W277.csv', 'set2_W278.csv', 'set2_W279.csv', 'set2_W28.csv', 'set2_W280.csv', 'set2_W281.csv', 'set2_W282.csv', 'set2_W29.csv', 'set2_W3.csv', 'set2_W30.csv', 'set2_W31.csv', 'set2_W32.csv', 'set2_W33.csv', 'set2_W34.csv', 'set2_W35.csv', 'set2_W36.csv', 'set2_W37.csv', 'set2_W38.csv', 'set2_W39.csv', 'set2_W4.csv', 'set2_W40.csv', 'set2_W41.csv', 'set2_W42.csv', 'set2_W43.csv', 'set2_W44.csv', 'set2_W45.csv', 'set2_W46.csv', 'set2_W47.csv', 'set2_W48.csv', 'set2_W49.csv', 'set2_W5.csv', 'set2_W50.csv', 'set2_W51.csv', 'set2_W52.csv', 'set2_W53.csv', 'set2_W54.csv', 'set2_W55.csv', 'set2_W56.csv', 'set2_W57.csv', 'set2_W58.csv', 'set2_W59.csv', 'set2_W6.csv', 'set2_W60.csv', 'set2_W61.csv', 'set2_W62.csv', 'set2_W63.csv', 'set2_W64.csv', 'set2_W65.csv', 'set2_W66.csv', 'set2_W67.csv', 'set2_W68.csv', 'set2_W69.csv', 'set2_W7.csv', 'set2_W70.csv', 'set2_W71.csv', 'set2_W72.csv', 'set2_W73.csv', 'set2_W74.csv', 'set2_W75.csv', 'set2_W76.csv', 'set2_W77.csv', 'set2_W78.csv', 'set2_W79.csv', 'set2_W8.csv', 'set2_W80.csv', 'set2_W81.csv', 'set2_W82.csv', 'set2_W83.csv', 'set2_W84.csv', 'set2_W85.csv', 'set2_W86.csv', 'set2_W87.csv', 'set2_W88.csv', 'set2_W89.csv', 'set2_W9.csv', 'set2_W90.csv', 'set2_W91.csv', 'set2_W92.csv', 'set2_W93.csv', 'set2_W94.csv', 'set2_W95.csv', 'set2_W96.csv', 'set2_W97.csv', 'set2_W98.csv', 'set2_W99.csv', 'Set2_Wounding1.csv', 'Set2_Wounding10.csv', 'Set2_Wounding100.csv', 'Set2_Wounding101.csv', 'Set2_Wounding102.csv', 'Set2_Wounding103.csv', 'Set2_Wounding104.csv', 'Set2_Wounding105.csv', 'Set2_Wounding106.csv', 'Set2_Wounding107.csv', 'Set2_Wounding108.csv', 'Set2_Wounding109.csv', 'Set2_Wounding11.csv', 'Set2_Wounding110.csv', 'Set2_Wounding111.csv', 'Set2_Wounding112.csv', 'Set2_Wounding113.csv', 'Set2_Wounding114.csv', 'Set2_Wounding115.csv', 'Set2_Wounding116.csv', 'Set2_Wounding117.csv', 'Set2_Wounding118.csv', 'Set2_Wounding119.csv', 'Set2_Wounding12.csv', 'Set2_Wounding120.csv', 'Set2_Wounding121.csv', 'Set2_Wounding122.csv', 'Set2_Wounding123.csv', 'Set2_Wounding124.csv', 'Set2_Wounding125.csv', 'Set2_Wounding126.csv', 'Set2_Wounding127.csv', 'Set2_Wounding128.csv', 'Set2_Wounding129.csv', 'Set2_Wounding13.csv', 'Set2_Wounding130.csv', 'Set2_Wounding131.csv', 'Set2_Wounding132.csv', 'Set2_Wounding133.csv', 'Set2_Wounding134.csv', 'Set2_Wounding135.csv', 'Set2_Wounding136.csv', 'Set2_Wounding137.csv', 'Set2_Wounding138.csv', 'Set2_Wounding139.csv', 'Set2_Wounding14.csv', 'Set2_Wounding140.csv', 'Set2_Wounding141.csv', 'Set2_Wounding142.csv', 'Set2_Wounding143.csv', 'Set2_Wounding144.csv', 'Set2_Wounding145.csv', 'Set2_Wounding146.csv', 'Set2_Wounding147.csv', 'Set2_Wounding148.csv', 'Set2_Wounding149.csv', 'Set2_Wounding15.csv', 'Set2_Wounding150.csv', 'Set2_Wounding151.csv', 'Set2_Wounding152.csv', 'Set2_Wounding153.csv', 'Set2_Wounding154.csv', 'Set2_Wounding155.csv', 'Set2_Wounding156.csv', 'Set2_Wounding157.csv', 'Set2_Wounding158.csv', 'Set2_Wounding159.csv', 'Set2_Wounding16.csv', 'Set2_Wounding160.csv', 'Set2_Wounding161.csv', 'Set2_Wounding162.csv', 'Set2_Wounding163.csv', 'Set2_Wounding164.csv', 'Set2_Wounding165.csv', 'Set2_Wounding166.csv', 'Set2_Wounding167.csv', 'Set2_Wounding168.csv', 'Set2_Wounding169.csv', 'Set2_Wounding17.csv', 'Set2_Wounding170.csv', 'Set2_Wounding171.csv', 'Set2_Wounding172.csv', 'Set2_Wounding173.csv', 'Set2_Wounding174.csv', 'Set2_Wounding175.csv', 'Set2_Wounding176.csv', 'Set2_Wounding177.csv', 'Set2_Wounding178.csv', 'Set2_Wounding179.csv', 'Set2_Wounding18.csv', 'Set2_Wounding180.csv', 'Set2_Wounding181.csv', 'Set2_Wounding182.csv', 'Set2_Wounding183.csv', 'Set2_Wounding184.csv', 'Set2_Wounding185.csv', 'Set2_Wounding186.csv', 'Set2_Wounding187.csv', 'Set2_Wounding188.csv', 'Set2_Wounding189.csv', 'Set2_Wounding19.csv', 'Set2_Wounding190.csv', 'Set2_Wounding191.csv', 'Set2_Wounding192.csv', 'Set2_Wounding193.csv', 'Set2_Wounding194.csv', 'Set2_Wounding195.csv', 'Set2_Wounding196.csv', 'Set2_Wounding197.csv', 'Set2_Wounding198.csv', 'Set2_Wounding199.csv', 'Set2_Wounding2.csv', 'Set2_Wounding20.csv', 'Set2_Wounding200.csv', 'Set2_Wounding201.csv', 'Set2_Wounding202.csv', 'Set2_Wounding203.csv', 'Set2_Wounding204.csv', 'Set2_Wounding205.csv', 'Set2_Wounding206.csv', 'Set2_Wounding207.csv', 'Set2_Wounding208.csv', 'Set2_Wounding209.csv', 'Set2_Wounding21.csv', 'Set2_Wounding210.csv', 'Set2_Wounding211.csv', 'Set2_Wounding212.csv', 'Set2_Wounding213.csv', 'Set2_Wounding214.csv', 'Set2_Wounding215.csv', 'Set2_Wounding216.csv', 'Set2_Wounding217.csv', 'Set2_Wounding218.csv', 'Set2_Wounding219.csv', 'Set2_Wounding22.csv', 'Set2_Wounding220.csv', 'Set2_Wounding221.csv', 'Set2_Wounding222.csv', 'Set2_Wounding223.csv', 'Set2_Wounding224.csv', 'Set2_Wounding225.csv', 'Set2_Wounding226.csv', 'Set2_Wounding227.csv', 'Set2_Wounding228.csv', 'Set2_Wounding229.csv', 'Set2_Wounding23.csv', 'Set2_Wounding230.csv', 'Set2_Wounding231.csv', 'Set2_Wounding232.csv', 'Set2_Wounding233.csv', 'Set2_Wounding234.csv', 'Set2_Wounding235.csv', 'Set2_Wounding236.csv', 'Set2_Wounding237.csv', 'Set2_Wounding238.csv', 'Set2_Wounding239.csv', 'Set2_Wounding24.csv', 'Set2_Wounding240.csv', 'Set2_Wounding241.csv', 'Set2_Wounding242.csv', 'Set2_Wounding243.csv', 'Set2_Wounding244.csv', 'Set2_Wounding245.csv', 'Set2_Wounding246.csv', 'Set2_Wounding247.csv', 'Set2_Wounding248.csv', 'Set2_Wounding249.csv', 'Set2_Wounding25.csv', 'Set2_Wounding250.csv', 'Set2_Wounding251.csv', 'Set2_Wounding252.csv', 'Set2_Wounding253.csv', 'Set2_Wounding254.csv', 'Set2_Wounding255.csv', 'Set2_Wounding256.csv', 'Set2_Wounding257.csv', 'Set2_Wounding258.csv', 'Set2_Wounding259.csv', 'Set2_Wounding26.csv', 'Set2_Wounding260.csv', 'Set2_Wounding261.csv', 'Set2_Wounding262.csv', 'Set2_Wounding263.csv', 'Set2_Wounding264.csv', 'Set2_Wounding265.csv', 'Set2_Wounding266.csv', 'Set2_Wounding267.csv', 'Set2_Wounding268.csv', 'Set2_Wounding269.csv', 'Set2_Wounding27.csv', 'Set2_Wounding270.csv', 'Set2_Wounding271.csv', 'Set2_Wounding272.csv', 'Set2_Wounding273.csv', 'Set2_Wounding274.csv', 'Set2_Wounding275.csv', 'Set2_Wounding276.csv', 'Set2_Wounding277.csv', 'Set2_Wounding278.csv', 'Set2_Wounding279.csv', 'Set2_Wounding28.csv', 'Set2_Wounding280.csv', 'Set2_Wounding281.csv', 'Set2_Wounding282.csv', 'Set2_Wounding29.csv', 'Set2_Wounding3.csv', 'Set2_Wounding30.csv', 'Set2_Wounding31.csv', 'Set2_Wounding32.csv', 'Set2_Wounding33.csv', 'Set2_Wounding34.csv', 'Set2_Wounding35.csv', 'Set2_Wounding36.csv', 'Set2_Wounding37.csv', 'Set2_Wounding38.csv', 'Set2_Wounding39.csv', 'Set2_Wounding4.csv', 'Set2_Wounding40.csv', 'Set2_Wounding41.csv', 'Set2_Wounding42.csv', 'Set2_Wounding43.csv', 'Set2_Wounding44.csv', 'Set2_Wounding45.csv', 'Set2_Wounding46.csv', 'Set2_Wounding47.csv', 'Set2_Wounding48.csv', 'Set2_Wounding49.csv', 'Set2_Wounding5.csv', 'Set2_Wounding50.csv', 'Set2_Wounding51.csv', 'Set2_Wounding52.csv', 'Set2_Wounding53.csv', 'Set2_Wounding54.csv', 'Set2_Wounding55.csv', 'Set2_Wounding56.csv', 'Set2_Wounding57.csv', 'Set2_Wounding58.csv', 'Set2_Wounding59.csv', 'Set2_Wounding6.csv', 'Set2_Wounding60.csv', 'Set2_Wounding61.csv', 'Set2_Wounding62.csv', 'Set2_Wounding63.csv', 'Set2_Wounding64.csv', 'Set2_Wounding65.csv', 'Set2_Wounding66.csv', 'Set2_Wounding67.csv', 'Set2_Wounding68.csv', 'Set2_Wounding69.csv', 'Set2_Wounding7.csv', 'Set2_Wounding70.csv', 'Set2_Wounding71.csv', 'Set2_Wounding72.csv', 'Set2_Wounding73.csv', 'Set2_Wounding74.csv', 'Set2_Wounding75.csv', 'Set2_Wounding76.csv', 'Set2_Wounding77.csv', 'Set2_Wounding78.csv', 'Set2_Wounding79.csv', 'Set2_Wounding8.csv', 'Set2_Wounding80.csv', 'Set2_Wounding81.csv', 'Set2_Wounding82.csv', 'Set2_Wounding83.csv', 'Set2_Wounding84.csv', 'Set2_Wounding85.csv', 'Set2_Wounding86.csv', 'Set2_Wounding87.csv', 'Set2_Wounding88.csv', 'Set2_Wounding89.csv', 'Set2_Wounding9.csv', 'Set2_Wounding90.csv', 'Set2_Wounding91.csv', 'Set2_Wounding92.csv', 'Set2_Wounding93.csv', 'Set2_Wounding94.csv', 'Set2_Wounding95.csv', 'Set2_Wounding96.csv', 'Set2_Wounding97.csv', 'Set2_Wounding98.csv', 'Set2_Wounding99.csv', 'w-100_pvals.csv', 'w-101_pvals.csv', 'w-102_pvals.csv', 'w-103_pvals.csv', 'w-104_pvals.csv', 'w-105_pvals.csv', 'w-106_pvals.csv', 'w-107_pvals.csv', 'w-108_pvals.csv', 'w-109_pvals.csv', 'w-10_pvals.csv', 'w-110_pvals.csv', 'w-111_pvals.csv', 'w-112_pvals.csv', 'w-113_pvals.csv', 'w-114_pvals.csv', 'w-115_pvals.csv', 'w-116_pvals.csv', 'w-117_pvals.csv', 'w-118_pvals.csv', 'w-119_pvals.csv', 'w-11_pvals.csv', 'w-120_pvals.csv', 'w-121_pvals.csv', 'w-122_pvals.csv', 'w-123_pvals.csv', 'w-124_pvals.csv', 'w-125_pvals.csv', 'w-126_pvals.csv', 'w-127_pvals.csv', 'w-128_pvals.csv', 'w-129_pvals.csv', 'w-12_pvals.csv', 'w-130_pvals.csv', 'w-131_pvals.csv', 'w-132_pvals.csv', 'w-133_pvals.csv', 'w-134_pvals.csv', 'w-135_pvals.csv', 'w-136_pvals.csv', 'w-137_pvals.csv', 'w-138_pvals.csv', 'w-139_pvals.csv', 'w-13_pvals.csv', 'w-140_pvals.csv', 'w-141_pvals.csv', 'w-142_pvals.csv', 'w-143_pvals.csv', 'w-144_pvals.csv', 'w-145_pvals.csv', 'w-146_pvals.csv', 'w-147_pvals.csv', 'w-148_pvals.csv', 'w-149_pvals.csv', 'w-14_pvals.csv', 'w-150_pvals.csv', 'w-151_pvals.csv', 'w-152_pvals.csv', 'w-153_pvals.csv', 'w-154_pvals.csv', 'w-155_pvals.csv', 'w-156_pvals.csv', 'w-157_pvals.csv', 'w-158_pvals.csv', 'w-159_pvals.csv', 'w-15_pvals.csv', 'w-160_pvals.csv', 'w-161_pvals.csv', 'w-162_pvals.csv', 'w-163_pvals.csv', 'w-164_pvals.csv', 'w-165_pvals.csv', 'w-166_pvals.csv', 'w-167_pvals.csv', 'w-168_pvals.csv', 'w-169_pvals.csv', 'w-16_pvals.csv', 'w-170_pvals.csv', 'w-171_pvals.csv', 'w-172_pvals.csv', 'w-173_pvals.csv', 'w-174_pvals.csv', 'w-175_pvals.csv', 'w-176_pvals.csv', 'w-177_pvals.csv', 'w-178_pvals.csv', 'w-179_pvals.csv', 'w-17_pvals.csv', 'w-180_pvals.csv', 'w-181_pvals.csv', 'w-182_pvals.csv', 'w-183_pvals.csv', 'w-184_pvals.csv', 'w-185_pvals.csv', 'w-186_pvals.csv', 'w-187_pvals.csv', 'w-188_pvals.csv', 'w-189_pvals.csv', 'w-18_pvals.csv', 'w-190_pvals.csv', 'w-191_pvals.csv', 'w-192_pvals.csv', 'w-193_pvals.csv', 'w-194_pvals.csv', 'w-195_pvals.csv', 'w-196_pvals.csv', 'w-197_pvals.csv', 'w-198_pvals.csv', 'w-199_pvals.csv', 'w-19_pvals.csv', 'w-1_pvals.csv', 'w-200_pvals.csv', 'w-201_pvals.csv', 'w-202_pvals.csv', 'w-203_pvals.csv', 'w-204_pvals.csv', 'w-205_pvals.csv', 'w-206_pvals.csv', 'w-207_pvals.csv', 'w-208_pvals.csv', 'w-209_pvals.csv', 'w-20_pvals.csv', 'w-210_pvals.csv', 'w-211_pvals.csv', 'w-212_pvals.csv', 'w-213_pvals.csv', 'w-214_pvals.csv', 'w-215_pvals.csv', 'w-216_pvals.csv', 'w-217_pvals.csv', 'w-218_pvals.csv', 'w-219_pvals.csv', 'w-21_pvals.csv', 'w-220_pvals.csv', 'w-221_pvals.csv', 'w-222_pvals.csv', 'w-223_pvals.csv', 'w-224_pvals.csv', 'w-225_pvals.csv', 'w-226_pvals.csv', 'w-227_pvals.csv', 'w-228_pvals.csv', 'w-229_pvals.csv', 'w-22_pvals.csv', 'w-230_pvals.csv', 'w-231_pvals.csv', 'w-232_pvals.csv', 'w-233_pvals.csv', 'w-234_pvals.csv', 'w-235_pvals.csv', 'w-236_pvals.csv', 'w-237_pvals.csv', 'w-238_pvals.csv', 'w-239_pvals.csv', 'w-23_pvals.csv', 'w-240_pvals.csv', 'w-241_pvals.csv', 'w-242_pvals.csv', 'w-243_pvals.csv', 'w-244_pvals.csv', 'w-245_pvals.csv', 'w-246_pvals.csv', 'w-247_pvals.csv', 'w-248_pvals.csv', 'w-249_pvals.csv', 'w-24_pvals.csv', 'w-250_pvals.csv', 'w-251_pvals.csv', 'w-252_pvals.csv', 'w-253_pvals.csv', 'w-254_pvals.csv', 'w-255_pvals.csv', 'w-256_pvals.csv', 'w-257_pvals.csv', 'w-258_pvals.csv', 'w-259_pvals.csv', 'w-25_pvals.csv', 'w-260_pvals.csv', 'w-261_pvals.csv', 'w-262_pvals.csv', 'w-263_pvals.csv', 'w-264_pvals.csv', 'w-265_pvals.csv', 'w-266_pvals.csv', 'w-267_pvals.csv', 'w-268_pvals.csv', 'w-269_pvals.csv', 'w-26_pvals.csv', 'w-270_pvals.csv', 'w-271_pvals.csv', 'w-272_pvals.csv', 'w-273_pvals.csv', 'w-274_pvals.csv', 'w-275_pvals.csv', 'w-276_pvals.csv', 'w-277_pvals.csv', 'w-278_pvals.csv', 'w-279_pvals.csv', 'w-27_pvals.csv', 'w-280_pvals.csv', 'w-281_pvals.csv', 'w-282_pvals.csv', 'w-28_pvals.csv', 'w-29_pvals.csv', 'w-2_pvals.csv', 'w-30_pvals.csv', 'w-31_pvals.csv', 'w-32_pvals.csv', 'w-33_pvals.csv', 'w-34_pvals.csv', 'w-35_pvals.csv', 'w-36_pvals.csv', 'w-37_pvals.csv', 'w-38_pvals.csv', 'w-39_pvals.csv', 'w-3_pvals.csv', 'w-40_pvals.csv', 'w-41_pvals.csv', 'w-42_pvals.csv', 'w-43_pvals.csv', 'w-44_pvals.csv', 'w-45_pvals.csv', 'w-46_pvals.csv', 'w-47_pvals.csv', 'w-48_pvals.csv', 'w-49_pvals.csv', 'w-4_pvals.csv', 'w-50_pvals.csv', 'w-51_pvals.csv', 'w-52_pvals.csv', 'w-53_pvals.csv', 'w-54_pvals.csv', 'w-55_pvals.csv', 'w-56_pvals.csv', 'w-57_pvals.csv', 'w-58_pvals.csv', 'w-59_pvals.csv', 'w-5_pvals.csv', 'w-60_pvals.csv', 'w-61_pvals.csv', 'w-62_pvals.csv', 'w-63_pvals.csv', 'w-64_pvals.csv', 'w-65_pvals.csv', 'w-66_pvals.csv', 'w-67_pvals.csv', 'w-68_pvals.csv', 'w-69_pvals.csv', 'w-6_pvals.csv', 'w-70_pvals.csv', 'w-71_pvals.csv', 'w-72_pvals.csv', 'w-73_pvals.csv', 'w-74_pvals.csv', 'w-75_pvals.csv', 'w-76_pvals.csv', 'w-77_pvals.csv', 'w-78_pvals.csv', 'w-79_pvals.csv', 'w-7_pvals.csv', 'w-80_pvals.csv', 'w-81_pvals.csv', 'w-82_pvals.csv', 'w-83_pvals.csv', 'w-84_pvals.csv', 'w-85_pvals.csv', 'w-86_pvals.csv', 'w-87_pvals.csv', 'w-88_pvals.csv', 'w-89_pvals.csv', 'w-8_pvals.csv', 'w-90_pvals.csv', 'w-91_pvals.csv', 'w-92_pvals.csv', 'w-93_pvals.csv', 'w-94_pvals.csv', 'w-95_pvals.csv', 'w-96_pvals.csv', 'w-97_pvals.csv', 'w-98_pvals.csv', 'w-99_pvals.csv', 'w-9_pvals.csv', 'Wounding Summary-set2-pscore6.csv', 'Wounding Summary-set2.csv']\n"
     ]
    }
   ],
   "source": [
    "glob.os.chdir(\"d:/new GWAS results-12232020/set2-wounding/\")\n",
    "print(glob.os.listdir())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv(\"Wounding Summary-set2.csv\") # Read in files\n",
    "idx = df.score >= 6  # Rows in which the score is greater than 6 - save those\n",
    "df = df[idx]\n",
    "start = df[\"pos\"] - 10000 # Calculate the left border of LD window\n",
    "df['start'] = start # Add a column of the left border position\n",
    "end = df[\"pos\"] + 10000 # Calculate the right border of LD window\n",
    "df['end'] = end # Add a column of the right border position\n",
    "output_file_name =  \"Wounding Summary-set2-pscore6.csv\"\n",
    "df.to_csv(output_file_name) # Export new generated files using a new file name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Chromosome</th>\n",
       "      <th>Start</th>\n",
       "      <th>End</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Gene annotation 1</th>\n",
       "      <th>Gene annotation 2</th>\n",
       "      <th>Gene annotation 3</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>3760</td>\n",
       "      <td>5630</td>\n",
       "      <td>AT1G01010</td>\n",
       "      <td>NAC domain containing protein 1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NAC domain containing protein 1 (NAC001); FUNC...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>6915</td>\n",
       "      <td>8666</td>\n",
       "      <td>AT1G01020</td>\n",
       "      <td>Arv1-like protein</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ARV1; CONTAINS InterPro DOMAIN/s: Arv1-like pr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>7315</td>\n",
       "      <td>8666</td>\n",
       "      <td>AT1G01020</td>\n",
       "      <td>Arv1-like protein</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ARV1; CONTAINS InterPro DOMAIN/s: Arv1-like pr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>1</td>\n",
       "      <td>11864</td>\n",
       "      <td>12940</td>\n",
       "      <td>AT1G01030</td>\n",
       "      <td>AP2/B3-like transcriptional factor family protein</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NGATHA3 (NGA3); CONTAINS InterPro DOMAIN/s: Tr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>1</td>\n",
       "      <td>23519</td>\n",
       "      <td>31079</td>\n",
       "      <td>AT1G01040</td>\n",
       "      <td>dicer-like 1</td>\n",
       "      <td>Encodes a Dicer homolog. Dicer is a RNA helica...</td>\n",
       "      <td>dicer-like 1 (DCL1); CONTAINS InterPro DOMAIN/...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35171</td>\n",
       "      <td>5</td>\n",
       "      <td>26961224</td>\n",
       "      <td>26963614</td>\n",
       "      <td>AT5G67610</td>\n",
       "      <td>Uncharacterized conserved protein (DUF2215)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Uncharacterized conserved protein (DUF2215); F...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35172</td>\n",
       "      <td>5</td>\n",
       "      <td>26961224</td>\n",
       "      <td>26963614</td>\n",
       "      <td>AT5G67610</td>\n",
       "      <td>Uncharacterized conserved protein (DUF2215)</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Uncharacterized conserved protein (DUF2215); F...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35173</td>\n",
       "      <td>5</td>\n",
       "      <td>26964891</td>\n",
       "      <td>26965720</td>\n",
       "      <td>AT5G67620</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>unknown protein; FUNCTIONS IN: molecular_funct...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35174</td>\n",
       "      <td>5</td>\n",
       "      <td>26967535</td>\n",
       "      <td>26969306</td>\n",
       "      <td>AT5G67630</td>\n",
       "      <td>P-loop containing nucleoside triphosphate hydr...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>P-loop containing nucleoside triphosphate hydr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35175</td>\n",
       "      <td>5</td>\n",
       "      <td>26969546</td>\n",
       "      <td>26970548</td>\n",
       "      <td>AT5G67640</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>unknown protein; Has 1807 Blast hits to 1807 p...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>35176 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       Chromosome     Start       End       Gene  \\\n",
       "0               1      3760      5630  AT1G01010   \n",
       "1               1      6915      8666  AT1G01020   \n",
       "2               1      7315      8666  AT1G01020   \n",
       "3               1     11864     12940  AT1G01030   \n",
       "4               1     23519     31079  AT1G01040   \n",
       "...           ...       ...       ...        ...   \n",
       "35171           5  26961224  26963614  AT5G67610   \n",
       "35172           5  26961224  26963614  AT5G67610   \n",
       "35173           5  26964891  26965720  AT5G67620   \n",
       "35174           5  26967535  26969306  AT5G67630   \n",
       "35175           5  26969546  26970548  AT5G67640   \n",
       "\n",
       "                                       Gene annotation 1  \\\n",
       "0                        NAC domain containing protein 1   \n",
       "1                                      Arv1-like protein   \n",
       "2                                      Arv1-like protein   \n",
       "3      AP2/B3-like transcriptional factor family protein   \n",
       "4                                           dicer-like 1   \n",
       "...                                                  ...   \n",
       "35171        Uncharacterized conserved protein (DUF2215)   \n",
       "35172        Uncharacterized conserved protein (DUF2215)   \n",
       "35173                                                NaN   \n",
       "35174  P-loop containing nucleoside triphosphate hydr...   \n",
       "35175                                                NaN   \n",
       "\n",
       "                                       Gene annotation 2  \\\n",
       "0                                                    NaN   \n",
       "1                                                    NaN   \n",
       "2                                                    NaN   \n",
       "3                                                    NaN   \n",
       "4      Encodes a Dicer homolog. Dicer is a RNA helica...   \n",
       "...                                                  ...   \n",
       "35171                                                NaN   \n",
       "35172                                                NaN   \n",
       "35173                                                NaN   \n",
       "35174                                                NaN   \n",
       "35175                                                NaN   \n",
       "\n",
       "                                       Gene annotation 3  \n",
       "0      NAC domain containing protein 1 (NAC001); FUNC...  \n",
       "1      ARV1; CONTAINS InterPro DOMAIN/s: Arv1-like pr...  \n",
       "2      ARV1; CONTAINS InterPro DOMAIN/s: Arv1-like pr...  \n",
       "3      NGATHA3 (NGA3); CONTAINS InterPro DOMAIN/s: Tr...  \n",
       "4      dicer-like 1 (DCL1); CONTAINS InterPro DOMAIN/...  \n",
       "...                                                  ...  \n",
       "35171  Uncharacterized conserved protein (DUF2215); F...  \n",
       "35172  Uncharacterized conserved protein (DUF2215); F...  \n",
       "35173  unknown protein; FUNCTIONS IN: molecular_funct...  \n",
       "35174  P-loop containing nucleoside triphosphate hydr...  \n",
       "35175  unknown protein; Has 1807 Blast hits to 1807 p...  \n",
       "\n",
       "[35176 rows x 7 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df1 = pd.read_csv(\"Reference gene list.csv\") # Read the reference gene information file containing all genes in Arabidopsis\n",
    "df1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chr</th>\n",
       "      <th>pos</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>score</th>\n",
       "      <th>maf</th>\n",
       "      <th>mac</th>\n",
       "      <th>GVE</th>\n",
       "      <th>exp_ID</th>\n",
       "      <th>position</th>\n",
       "      <th>lipid name</th>\n",
       "      <th>lipid number</th>\n",
       "      <th>transformation</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>10813779</td>\n",
       "      <td>10803779</td>\n",
       "      <td>10823779</td>\n",
       "      <td>6.797301</td>\n",
       "      <td>0.155556</td>\n",
       "      <td>56</td>\n",
       "      <td>0.073963</td>\n",
       "      <td>set2_C1</td>\n",
       "      <td>2_10813779</td>\n",
       "      <td>acDGDG(18:4-O/34:8-2O)</td>\n",
       "      <td>lipid0601</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "      <td>8493505</td>\n",
       "      <td>8483505</td>\n",
       "      <td>8503505</td>\n",
       "      <td>7.081350</td>\n",
       "      <td>0.219444</td>\n",
       "      <td>79</td>\n",
       "      <td>0.077236</td>\n",
       "      <td>set2_C1</td>\n",
       "      <td>3_8493505</td>\n",
       "      <td>acDGDG(18:4-O/34:8-2O)</td>\n",
       "      <td>lipid0601</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "      <td>8493630</td>\n",
       "      <td>8483630</td>\n",
       "      <td>8503630</td>\n",
       "      <td>6.826690</td>\n",
       "      <td>0.227778</td>\n",
       "      <td>82</td>\n",
       "      <td>0.074302</td>\n",
       "      <td>set2_C1</td>\n",
       "      <td>3_8493630</td>\n",
       "      <td>acDGDG(18:4-O/34:8-2O)</td>\n",
       "      <td>lipid0601</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>3</td>\n",
       "      <td>19291713</td>\n",
       "      <td>19281713</td>\n",
       "      <td>19301713</td>\n",
       "      <td>6.308182</td>\n",
       "      <td>0.086111</td>\n",
       "      <td>31</td>\n",
       "      <td>0.068311</td>\n",
       "      <td>set2_C1</td>\n",
       "      <td>3_19291713</td>\n",
       "      <td>acDGDG(18:4-O/34:8-2O)</td>\n",
       "      <td>lipid0601</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>5</td>\n",
       "      <td>25721644</td>\n",
       "      <td>25711644</td>\n",
       "      <td>25731644</td>\n",
       "      <td>8.602853</td>\n",
       "      <td>0.127778</td>\n",
       "      <td>46</td>\n",
       "      <td>0.094636</td>\n",
       "      <td>set2_C1</td>\n",
       "      <td>5_25721644</td>\n",
       "      <td>acDGDG(18:4-O/34:8-2O)</td>\n",
       "      <td>lipid0601</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1650</td>\n",
       "      <td>5</td>\n",
       "      <td>12188171</td>\n",
       "      <td>12178171</td>\n",
       "      <td>12198171</td>\n",
       "      <td>6.511004</td>\n",
       "      <td>0.094444</td>\n",
       "      <td>34</td>\n",
       "      <td>0.070657</td>\n",
       "      <td>set2_C94</td>\n",
       "      <td>5_12188171</td>\n",
       "      <td>DGDG(20:2/18:3)</td>\n",
       "      <td>lipid2020</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1651</td>\n",
       "      <td>5</td>\n",
       "      <td>12188214</td>\n",
       "      <td>12178214</td>\n",
       "      <td>12198214</td>\n",
       "      <td>6.700183</td>\n",
       "      <td>0.097222</td>\n",
       "      <td>35</td>\n",
       "      <td>0.072842</td>\n",
       "      <td>set2_C94</td>\n",
       "      <td>5_12188214</td>\n",
       "      <td>DGDG(20:2/18:3)</td>\n",
       "      <td>lipid2020</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1652</td>\n",
       "      <td>1</td>\n",
       "      <td>10125514</td>\n",
       "      <td>10115514</td>\n",
       "      <td>10135514</td>\n",
       "      <td>8.329319</td>\n",
       "      <td>0.100000</td>\n",
       "      <td>36</td>\n",
       "      <td>0.091525</td>\n",
       "      <td>set2_C95</td>\n",
       "      <td>1_10125514</td>\n",
       "      <td>DGDG(20:1/18:3)</td>\n",
       "      <td>lipid2021</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1653</td>\n",
       "      <td>1</td>\n",
       "      <td>10125515</td>\n",
       "      <td>10115515</td>\n",
       "      <td>10135515</td>\n",
       "      <td>8.652385</td>\n",
       "      <td>0.102778</td>\n",
       "      <td>37</td>\n",
       "      <td>0.095199</td>\n",
       "      <td>set2_C95</td>\n",
       "      <td>1_10125515</td>\n",
       "      <td>DGDG(20:1/18:3)</td>\n",
       "      <td>lipid2021</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1654</td>\n",
       "      <td>1</td>\n",
       "      <td>16645949</td>\n",
       "      <td>16635949</td>\n",
       "      <td>16655949</td>\n",
       "      <td>6.457952</td>\n",
       "      <td>0.100000</td>\n",
       "      <td>36</td>\n",
       "      <td>0.070044</td>\n",
       "      <td>set2_C98</td>\n",
       "      <td>1_16645949</td>\n",
       "      <td>DGMG(16:4-O)</td>\n",
       "      <td>lipid2023</td>\n",
       "      <td>BOXCOX</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1655 rows × 13 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      chr       pos     start       end     score       maf  mac       GVE  \\\n",
       "0       2  10813779  10803779  10823779  6.797301  0.155556   56  0.073963   \n",
       "1       3   8493505   8483505   8503505  7.081350  0.219444   79  0.077236   \n",
       "2       3   8493630   8483630   8503630  6.826690  0.227778   82  0.074302   \n",
       "3       3  19291713  19281713  19301713  6.308182  0.086111   31  0.068311   \n",
       "4       5  25721644  25711644  25731644  8.602853  0.127778   46  0.094636   \n",
       "...   ...       ...       ...       ...       ...       ...  ...       ...   \n",
       "1650    5  12188171  12178171  12198171  6.511004  0.094444   34  0.070657   \n",
       "1651    5  12188214  12178214  12198214  6.700183  0.097222   35  0.072842   \n",
       "1652    1  10125514  10115514  10135514  8.329319  0.100000   36  0.091525   \n",
       "1653    1  10125515  10115515  10135515  8.652385  0.102778   37  0.095199   \n",
       "1654    1  16645949  16635949  16655949  6.457952  0.100000   36  0.070044   \n",
       "\n",
       "        exp_ID    position              lipid name lipid number transformation  \n",
       "0      set2_C1  2_10813779  acDGDG(18:4-O/34:8-2O)    lipid0601         BOXCOX  \n",
       "1      set2_C1   3_8493505  acDGDG(18:4-O/34:8-2O)    lipid0601         BOXCOX  \n",
       "2      set2_C1   3_8493630  acDGDG(18:4-O/34:8-2O)    lipid0601         BOXCOX  \n",
       "3      set2_C1  3_19291713  acDGDG(18:4-O/34:8-2O)    lipid0601         BOXCOX  \n",
       "4      set2_C1  5_25721644  acDGDG(18:4-O/34:8-2O)    lipid0601         BOXCOX  \n",
       "...        ...         ...                     ...          ...            ...  \n",
       "1650  set2_C94  5_12188171         DGDG(20:2/18:3)    lipid2020         BOXCOX  \n",
       "1651  set2_C94  5_12188214         DGDG(20:2/18:3)    lipid2020         BOXCOX  \n",
       "1652  set2_C95  1_10125514         DGDG(20:1/18:3)    lipid2021         BOXCOX  \n",
       "1653  set2_C95  1_10125515         DGDG(20:1/18:3)    lipid2021         BOXCOX  \n",
       "1654  set2_C98  1_16645949            DGMG(16:4-O)    lipid2023         BOXCOX  \n",
       "\n",
       "[1655 rows x 13 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df2 = pd.read_csv(\"Wounding Summary-set2-pscore6.csv\") # Read the new generated GWAS summary file with start and end position of LD window for each SNP\n",
    "df2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0       ['AT2G25360', 'AT2G25370', 'AT2G25370', 'AT2G2...\n",
       "1       ['AT3G23630', 'AT3G23633', 'AT3G23633', 'AT3G2...\n",
       "2       ['AT3G23630', 'AT3G23633', 'AT3G23633', 'AT3G2...\n",
       "3       ['AT3G51960', 'AT3G51960', 'AT3G51970', 'AT3G5...\n",
       "4       ['AT5G64280', 'AT5G64290', 'AT5G64300', 'AT5G6...\n",
       "                              ...                        \n",
       "1650                                                   []\n",
       "1651                                                   []\n",
       "1652    ['AT1G29010', 'AT1G29020', 'AT1G29025', 'AT1G2...\n",
       "1653    ['AT1G29010', 'AT1G29020', 'AT1G29025', 'AT1G2...\n",
       "1654    ['AT1G43890', 'AT1G43890', 'AT1G43890', 'AT1G4...\n",
       "Name: Gene, Length: 1655, dtype: object"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "LD = [] #Generate an empty list\n",
    "array = [] #Generate an empty array\n",
    "for i in range(len(df2.index)): #Pick up each SNP \n",
    "    for j in range(len(df1.index)): #With each SNP, loop up all nearby genes within LD window\n",
    "        if df1.loc[j,\"Chromosome\"] == df2.loc[i,\"chr\"] and df2.loc[i,'end']>=df1.loc[j,'Start'] and df2.loc[i,'start']<=df1.loc[j,'End']:\n",
    "            gene_ID = df1.loc[j,'Gene']\n",
    "            LD.append(gene_ID) #Generate a candidate list for each SNP\n",
    "    array.append(str(LD)) #Generate a candidate array for all significant SNPs\n",
    "    LD = []\n",
    "df2['Gene'] = array\n",
    "df2['Gene'] #Add a new column with candidate gene list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "df2.to_csv(\"Wounding-set2-pscore6-candidate gene list.csv\") # Export new generated files using a new file name"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
